worker_test.go

```diff
@@ -118,8 +118,8 @@ func TestWorkerGetOpt(t *testing.T) {
 
 func ExampleServeHTTP_workers() {
 	if err := frankenphp.Init(
-		frankenphp.WithWorkers("worker1.php", 4, map[string]string{"ENV1": "foo"}, []string{}),
-		frankenphp.WithWorkers("worker2.php", 2, map[string]string{"ENV2": "bar"}, []string{}),
+		frankenphp.WithWorkers("worker1", "worker1.php", 4, map[string]string{"ENV1": "foo"}, []string{}),
+		frankenphp.WithWorkers("worker2", "worker2.php", 2, map[string]string{"ENV2": "bar"}, []string{}),
```

Comment:
It may be good to add a test for when workers have the same name.

Reply:
test added.

Caddy does not allow [duplicate server name](https://github.com/caddyserver/caddy/blob/master/caddyconfig/httpcaddyfile/serveroptions.go#L282-L284) and FrankenPHP allows duplicate `worker.file` hence we can't validate if the worker has unique name.

in this case, only single metrics will be used here

```
{
	frankenphp {
		worker {
			file ./testdata/index.php
			name service1
			num 3
		}
		worker {
		 	file ./svc2/index.php
		 	name service1
			num 2
		}
	}

```

the metrics will be like this

```
frankenphp_ready_workers{worker="service1"} 5
```

Reply:
perfect! Uniqueness isn't needed, but we also want to be sure it doesn't crash.